CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 1.1.0 / 2016-09-01
+
+* [CHANGE] Kubernetes SD: Drop `__meta_kubernetes_role`.
```

Comment:
Good catch @brian-brazil! This changelog entry is wrong. We did not drop the meta label, that's still there afaik: https://github.com/prometheus/prometheus/blob/master/retrieval/discovery/kubernetes/discovery.go#L38-L39

The commit in question just aimed at simplifying the example config https://github.com/prometheus/prometheus/commit/f97cd29